src/app/components/observation/summary/summary.component.html

```diff
@@ -22,7 +22,7 @@ <h3 *ngIf="!isNameSameAsPreviousName(i)">{{ summary.RegistrationName }}
     </ng-container>
   </ng-container>
   <ng-template #TextSummary>
-    <h4 *ngIf="showHeaders">{{ header }}:&nbsp;</h4><span>{{ value }}</span>
+    <h4 *ngIf="showHeaders">{{ header }}:&nbsp;</h4><span class="text">{{ value }}</span>
```

Comment:
```suggestion
    <h4 *ngIf="showHeaders">{{ header }}:&nbsp;</h4><span class="text">{{ value }}</span><br>
```

Reply:
Jeg lurer på om ikke observasjonskortene blir veldig lange hvis vi skal ha linjeskift for hvert felt, men nå har jeg forsøkt å legge linjeskift foran verdier som inneholder linjeskift. Da blir det slik:
![image](https://user-images.githubusercontent.com/71138449/201693771-ffedef39-7d92-4a8b-957e-a286952f0e4a.png)

Reply:
Det løser ikke problemet.
![Skjermbilde 2022-11-14 161454](https://user-images.githubusercontent.com/12639272/201696287-d445fb52-cd76-4f8b-9de8-a3fadf59da67.jpg)

Reply:
Sant. Kommentarfelt burde vært vist på slutten av et skjema for at det skal bli lettere å lese, men jeg la inn en fiks nå som også legger til et linjeskift etter fler-linjes-tekster. 
Kan det funke?
![image](https://user-images.githubusercontent.com/71138449/201701582-05810a6f-74a9-4353-9838-eef0c1e59e83.png)